src/dsverifier.cpp

```diff
@@ -848,7 +891,7 @@ std::string prepare_bmc_command_line()
 	  command_line = "gcc -E " + desired_filename + " -DK_INDUCTION_MODE=K_INDUCTION -DBMC=ESBMC -I " + bmc_path;
 	} else {
           command_line = model_checker_path + "/esbmc " + desired_filename +
-	  " --no-bounds-check --no-pointer-check --no-div-by-zero-check -DBMC=ESBMC -I " +
```

Comment:
Sorry, I was trying to detect a division by zero in my code and I deleted it. I'm going to write it back.